libp2p/transports/transport.nim

```diff
@@ -91,17 +130,10 @@ method upgradeOutgoing*(
 
   self.upgrader.upgradeOutgoing(conn)
 
-method handles*(
-  self: Transport,
-  address: MultiAddress): bool {.base, gcsafe.} =
-  ## check if transport supports the multiaddress
-  ##
+method `==`*(
```

Comment:
Not sure why you're removing `handles`, if we want to have helper operator as it is in this case, we should just provide it as a wrapper to `handles`. One reason is making operators a `method`, I'm not even sure if that makes sense and other languages prohibit this explicitly.

Reply:
I think he just moved it above to be able to use it in start

Reply:
Yep, it was moved above, but I'll put it back since we don't need to do any filtering in the base class `start` (see comment below) :)

Reply:
I originally thought that this was replacing the `handles` method, but I realize now it's something else. Where in the tests are we using this?

Reply:
After removing `swithc.listenError` and associated tests, `transport.==` is no longer used in the tests, so it has been now been removed. Good catch!